generator/internal/language/rust.go

```diff
@@ -653,73 +653,6 @@ func rustBodyAccessor(m *api.Method) string {
 	return "." + rustToSnake(m.PathInfo.BodyFieldPath)
 }
 
-func rustHTTPPathFmt(m *api.PathInfo) string {
```

Comment:
Nice to see this go, but the golang code still looks like does most of the work in the codec?

Reply:
Yeah, I want to clean that up too. But I think I'll do it in another PR, WDYT?

Reply:
A separate PR SGTM.